### PR TITLE
Confirm clicking Reply while writing a comment

### LINKED ISSUE #7000: Confirmations Addon: Confirm when you press the "reply" button
### Why this would be helpful

When I am trying to write a comment, I sometimes accidentally press the "reply" button which removes the comment you are writing. It would be helpful to warn the user when there is content in the comment and you press the "reply" button.

### My suggestion

Warn the user about clicking the "reply" button when the comment has text in it and add the feature to the "Confirmations" addon. If the comment you are writing is empty, don't warn the user about clicking the "reply" button.

### Possible alternatives

_No response_

### Additional context

comment

### PR DESCRIPTION
**Pull request by DNin01**
Resolves #7000

https://github.com/ScratchAddons/ScratchAddons/assets/106490990/ae2df17a-fc3f-4d0a-97a6-645e59fa00c2

### Changes

Each comment on Scratch has a reply button that shows the comment box when clicked. If you click on the reply button again, the comment box is removed and your comment gets discarded. This update shows a confirmation before this action, if you've enabled "Confirm canceling comments", of course.

### Reason for changes

To prevent accidentally discarding comments.

### Tests

Tested on profile pages, projects, and studios, only a little bit.

### CODE REVIEW

**Review by mxmou — APPROVED** [review]

**Review by WorldLanguages — APPROVED** [review]
Nice! LGTM

**Comment by WorldLanguages** [comment]
I will merge #7017 after this one gets merged

**Review by Samq64 — APPROVED** [review]

**Pull request merged**
